docker/ubuntu/Dockerfile

```diff
@@ -30,6 +30,8 @@
 # 22.04 is the current long term support release, maintained until 2025-04.
 FROM ubuntu:22.04 AS os
 
+RUN apt-get update
```

Comment:
Why did you move this into a separate `RUN`? 🤔

Reply:
Changed it

Reply:
Where? :thinking: